src/ui/utils/user.ts

```diff
@@ -1,7 +1,13 @@
+import { Nag } from "ui/hooks/users";
+
 export function getAvatarColor(avatarID?: number) {
   if (!avatarID) {
     return "#696969";
   }
   const avatarColors = ["#2D4551", "#509A8F", "#E4C478", "#E9A56C", "#D97559"];
   return avatarColors[avatarID % avatarColors.length];
 }
+
+export function shouldShowNag(nags: Nag[], key: Nag) {
```

Comment:
What if we explicitly called the `nags` param `nagsToHide` or `dontShow`, or `hiddenNags`, just something to make it clear what the significance of `nags` is?

Reply:
`shouldShowNag(hiddenNags:` hurts my brain even more lol... 

Never sure how about OO in these contexts, but yeah an object is what i thought of first too